tensorflow/core/kernels/depthtospace_op.cc

```diff
@@ -34,12 +33,14 @@ limitations under the License.
 #include "tensorflow/core/platform/logging.h"
 #include "tensorflow/core/platform/types.h"
 #include "tensorflow/core/util/tensor_format.h"
+#include "third_party/eigen3/unsupported/Eigen/CXX11/Tensor"
```

Comment:
The order of includes changed according to clang-formatting tool with `-style=google`